packages/@okta/vuepress-site/docs/guides/custom-email/main/index.md

```diff
@@ -148,25 +133,25 @@ VTL syntax:
 
 You can also reference these org-level attributes, such as:
 
-* `$org.name`
-* `$org.locale`
-* `$org.subDomain`
+* `${org.name}`
+* `${org.locale}`
+* `${org.subDomain}`
 
-### Support for conditional logic
+## Use app context
 
 <ApiLifecycle access="ie" />
 
-All conditional logic that is supported by the Velocity Templating Engine, such as `if`, `elseif`, or `else` constructs and `foreach` loops, is available for you to use in your templates. See the [Velocity documentation](http://velocity.apache.org/engine/1.7/user-guide.html).
+Okta Identity Engine orgs have access to app context within emails using VTL. When an end user enters an authentication flow, Identity Engine stores the app context in the state token. The following properties are available in the app context:
 
-## Use app context
+* `${app.id}`
+* `${app.name}`
+* `${app.label}`
 
-Okta Identity Engine orgs have access to app context within emails using the Velocity Templating Language. When an end user enters an authentication flow, Identity Engine stores the app context in the state token. The following properties are available in the app context:
+When these properties are used with conditional logic, you can trigger branding for the specified app and define strings to uniquely customize an email template based on the app from where the email was triggered. App context is not available on Classic Engine since the state token does not exist there.
```

Comment:
```suggestion
When these properties are used with conditional logic, you can trigger branding for the specified app and define strings to uniquely customize an email template based on the app from where the email was triggered. App context isn't available on the Classic Engine since the state token doesn't exist there.
```